include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -527,40 +621,34 @@ private :
             std::cout << "Meridian segment crossing pole" << std::endl;
 #endif
             CT sign_non_zero = lat3 >= c0 ? 1 : -1;
-            result_distance_point_segment<CT> res13 =
-                    apply<geometry::radian>(lon1, lat1,
-                                            lon1, half_pi * sign_non_zero,
-                                            lon3, lat3, spheroid);
-            result_distance_point_segment<CT> res23 =
-                    apply<geometry::radian>(lon2, lat2,
-                                            lon2, half_pi * sign_non_zero,
-                                            lon3, lat3, spheroid);
-            if (res13.distance < res23.distance)
-            {
-                return res13;
-            }
-            else
-            {
-                return res23;
-            }
+            
+            auto res13 = apply(lon1, lat1, lon1, half_pi * sign_non_zero, lon3, 
+                lat3, spheroid);
```

Comment:
Yet another version of function call formatting

Reply:
one line now